Code/GraphMol/MolStandardize/catch_tests.cpp

```diff
@@ -1706,4 +1707,27 @@ TEST_CASE(
                   ValueErrorException);
   CHECK_THROWS_AS(MolStandardize::superParentInPlace(mols, numThreads),
                   ValueErrorException);
+}
+
+TEST_CASE("github #7689 RDKitValidation does not catch some valence issues") {
+  SECTION("basics") {
+    std::string mb = R"CTAB(foo
```

Comment:
This test already  gets flagged on rdkit, so it does not appear to be sufficient for the fix

```
>>> rdkit.__version__
'2024.03.5'
```

```
mb = """foo
  MJ240300                      

  2  1  0  0  0  0  0  0  0  0999 V2000
   -4.8993    1.8410    0.0000 Br  0  5  0  0  0  0  0  0  0  0  0  0
   -5.6138    1.4285    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
  2  1  1  0  0  0  0
M  END"""
>>> Chem.MolFromMolBlock(mb)
[13:02:54] Explicit valence for atom # 0 Br, 2, is greater than permitted
```

Reply:
Or is this coming from somewhere else?

Reply:
@bp-kelley I believe the isoelectronic valence changes in `2024.09.1b1` suppressed the warnings; they were indeed there in 2024.03.5. The test is against `2024.09.1b1`.

Reply:
Yes, this was a bug introduced with the isoelectronic valence changes

Reply:
Ok,  the regression was added here:  https://github.com/rdkit/rdkit/pull/7491

I'm up to speed.  Looks like a necessary fix!